### ISSUE TITLE
v ...20: Safari again

### ISSUE THREAD

**Issue by TeaMike** [XiozZe/XioScript#61]
i recall having seen this error in the past days with a previous version but not every time:

Failed to load resource: the server responded with a status of 500 (Internal Server Error)
its then blocking the fb frames and the usual long list of "Deprecated attempt to access property" like last time.

It took a few attempts executing the same Generate all ending up with "safari Content quit unexpectedly. This report will be sent to Apple automatically" and "A problem occurred with this webpage so it was reloaded" and no functions generated, until it finally worked (few times the screen reloaded is the http://virtonomics.com/mary/main/ page

Edit: until now, any script i ever installed worked equally well (or not) in chrome and safari. Should this be different with XS because of sophisticated script functionality it is of course acceptable to label the script made for chrome and not support safari.

Edit 2
- tested the same thing in Chrome. works like a charm there

**Comment by XiozZe** [comment]
With solving the Spanish languages issues on #60, I have repaired the error function of server calls. Originally it was meant to resend a server call in case it stumbled on an error is that, however, it was bugged, partly because it was never used as the English realm never had given me such an error. I guess the moment I scripted it was back in the period when the virtonomics server was not as great as that it is now. I felt that the server works much faster nowadays than that it did back then.

Try it again with XS 12.0.21 (not uploaded yet), maybe your problem is miraculously solved by then. Otherwise it might be the same problem Firefox has: the script is working too long and the browser get worried it might be stuck in an infinite loop and so it throws an error.
